whitesource/util.py

```diff
@@ -1,10 +1,167 @@
 import functools
 
+import tabulate
+import typing
+
 import ci.util
 import whitesource.client
+import mailutil
+import mail.model
 
 
 @functools.lru_cache()
 def create_whitesource_client(whitesource_cfg_name: str):
     cfg_fac = ci.util.ctx().cfg_factory()
     return whitesource.client.WhitesourceClient(cfg_fac.whitesource(whitesource_cfg_name))
+
+
+def generate_reporting_tables(report,
+                              threshold: float,
+                              tablefmt):
+
+    # monkeypatch: disable html escaping
+    tabulate.htmlescape = lambda x: x
+
+    # split respecting CVSS-V3 threshold
+    above = {}
+    below = {}
+
+    for lib, dic in report.items():
+        if float(dic['CVSS-V3']) > threshold:
```

Comment:
I still would have preferred to use a dataclass - or at least deduplicate the accessor-function returning the cvss-float :-/